cosmwasm/submit-proposal.js

```diff
@@ -250,12 +285,20 @@ const programHandler = () => {
 
     const executeCmd = program
         .command('execute')
-        .description('Submit a execute wasm contract proposal')
+        .description('Submit an execute wasm contract proposal')
         .action((options) => {
             mainProcessor(execute, options);
         });
     addAmplifierOptions(executeCmd, { contractOptions: true, executeProposalOptions: true, proposalOptions: true, runAs: true });
 
+    const registerItsChainCmd = program
+        .command('register-its-chain')
```

Comment:
```suggestion
        .command('its-hub-register-chains <chains...>')
```
maybe? prefix is the contract name, and suffix is ExecMsg type

Also, use positional arg instead of keyword arg since it's required